.goreleaser.yml

```diff
@@ -23,10 +23,15 @@ builds:
       - 386
       - arm
       - arm64
+      - mips
+      - mips64
```

Comment:
There are also the little-endian architectures `mipsle` and `mips64le`.

Reply:
Do we want to add a UPX compression step for MIPS binaries (see [example](https://github.com/goreleaser/goreleaser/issues/146#issuecomment-288793339))?

Typically, those chips run in embedded environment where available space is always critically low (and Go binaries are way to large).

(I'm also OK with leaving this as an exercise for the user).

Reply:
UPX have some side effect, I prefer to let user play with that.